yogadl/storage/_cloud_storage.py

```diff
@@ -33,13 +33,19 @@ class BaseCloudConfigurations(metaclass=abc.ABCMeta):
     """
 
     def __init__(
-        self, bucket: str, bucket_directory_path: str, url: str, local_cache_dir: str,
+        self,
+        bucket: str,
+        bucket_directory_path: str,
+        url: str,
+        local_cache_dir: str,
+        verify: Optional[str],
```

Comment:
could we rename this to something more self-explanatory, maybe `tls_certificate_path`?